test/event_test.rb

```diff
@@ -135,25 +135,32 @@ def test_attendees_returns_an_array_of_attendees_with_assigned_numbers
   def test_updates_event_with_params
     VCR.use_cassette 'event-update' do
       find_event
-      response = @event.update name: 'Movember 2014'
-      assert response, 'Should return true after updating'
+      event = @event.update 'event.name.html' => 'Movember 2014'
+      assert_kind_of Chiketto::Event, event
       @event = Chiketto::Event.find @event.id
       assert_equal 'Movember 2014', @event.name.to_s
     end
   end
 
   def test_update_returns_false_for_invalid_id
-    VCR.use_cassette 'event-update-fail' do
-      event = Chiketto::Event.new id: '12345'
-      response = event.update name: 'Movember 2014'
-      refute response, 'Should return false for invalid id'
+    VCR.use_cassette 'event-update-failure' do
+      assert_raises Chiketto::Exception do
+        event = Chiketto::Event.new id: '12345'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.